src/data/faq.json

```diff
@@ -1730,9 +1731,9 @@
                                 "anchor": "who_can_help",
                                 "active": false,
                                 "textblock": [
-                                    "<b>Updated January 18, 2023</b>",
+                                    "<b>Updated February 1, 2023</b>",
                                     "The technical hotline (<a href='#international_phone_numbers' target='_blank' >click here to see the phone numbers</a>) will help you with technical questions about the Corona-Warn-App, for instance if you have problems with exposure logging. You can reach the technical hotline from Monday to Friday from 7 a.m. to 5 p.m. (except on German national holidays), and it is free of charge for you within Germany.",
-                                    "The TAN hotline (<a href='#international_phone_numbers' target='_blank' >click here to see the phone numbers</a>) will help you to enter the TAN if you have a positive PCR test result and if you have not received a TAN or document with a PCR-Test QR code. The TAN hotline is available Monday to Sunday from 7 a.m. to 8 p.m. German local time. The call is free of charge from within Germany. Please note that as of version 3.0 of the Corona-Warn-App, a teleTAN is no longer required to share a positive test result that has not been transmitted to the Corona-Warn-App. Therefore, the operation of the TAN hotline will be discontinued as of January 31, 2023.",
+                                    "The operation of the TAN hotline has been discontinued as of January 31, 2023. Previously, it helped you enter the TAN if you had a positive PCR test result and if you did not receive a TAN or a document with a PCR test QR code. You could call them from Monday to Sunday from 7 a.m. to 8 p.m. local time in Germany. The call was free of charge within Germany. Please note that since version 3.0 of the Corona-Warn-App, teleTAN is no longer required to share a positive test result that was not submitted to the Corona-Warn-App.",

```

Comment:
Not really helpful to leave the operating hours and other details here.

Reply:
Updated in https://github.com/corona-warn-app/cwa-website/pull/3357/commits/b006857fcb5fdb081ce62f9f90250306044fa7f1, thanks for the suggestion!